client/app/scripts/superdesk-authoring/views/article-edit.html

```diff
@@ -69,13 +69,27 @@
         ng-change="autosave(item)"
         ng-if="_editable"
         ng-trim="false"></textarea>
-	<div class="abstract" ng-if="!_editable">{{item.description}}</div>
+  <div class="abstract" ng-if="!_editable">{{item.description}}</div>
 </div>
 
 <div class="field" ng-if="isMediaType">
   <div class="full-preview" ng-if="item.type == 'picture'" sd-ratio-calc>
-    <div class="rendition" sd-item-rendition data-item="item" data-ratio="ratio" data-rendition="baseImage"></div>
+    <div ng-if="toggleDetails" class="rendition" sd-item-rendition data-item="item" data-ratio="ratio" data-rendition="baseImage"></div>
+
+    <div id="dynBoxId1" class="item-detail-info" ng-class="{active: !toggleDetails}">
+      <div class="original">
+        <ul>
+          <li ng-repeat="cropsize in metadata.crop_sizes track by cropsize.name">
+          <label translate>ASPECT RATIO: {{cropsize.name.replace('-', '/')}}</label>
+          <div sd-image-crop ng-if="item.type === 'picture'" ng-click="editCrop(cropsize)" data-src="item.renditions.original.href" data-box-width="cropsize.width" data-box-height="cropsize.height" data-aspect-ratio="evalAspectRatio(cropsize.name)" data-crop-select="item.renditions[cropsize.name]" tooltip="{{ :: 'Edit Crop' | translate }}"></div>
```

Comment:
This line is looooong, better to wrap it to keep the viewport width under control (i.e. prevent horizontal scrollbar).